CHANGELOG.md

```diff
@@ -58,6 +58,9 @@ A brief description of the categories of changes:
 * (gazelle) Use relative paths if possible for dependencies added through
   the use of the `resolve` directive.
 
+* (gazelle) When using `python_generation_mode file`, one `py_test` target is
+  made per test file.
```

Comment:
Could you please clarify the CHANGELOG, that it is for cases where `__test__` exists?